package-lock.json

```diff
@@ -1,8 +1,13516 @@
 {
   "name": "handlebars",
-  "version": "4.7.6",
-  "lockfileVersion": 1,
+  "version": "4.7.7",
+  "lockfileVersion": 2,
```

Comment:
:information_source: `lockfileVersion` is backwards compatible to npm 6.

The latest supported Node.js version is v12, which ships npm 6.